src/cider_nrepl/main.clj

```diff
@@ -1,7 +1,7 @@
 (ns cider-nrepl.main
   (:require
-    [clojure.java.io :as io]
-    [clojure.tools.nrepl.server :as nrepl.server]))
+   [clojure.java.io :as io]
```

Comment:
Btw, this seems unrelated to your changes - how was the build passing before?

Reply:
emacs vs vim indentation I think.

Reply:
Cljfmt flagged the formatting of those lines. However cljfmt, Eastwood, and cloverage happen to be configured as allowed_failure in CI, so I assume someone’s editor is configured differently and that went unnoticed.

Reply:
Got it!